repl/src/main/scala/com/cloudera/livy/repl/Session.scala

```diff
@@ -168,4 +195,21 @@ class Session(interpreter: Interpreter, stateChangedCallback: SessionState => Un
 
     compact(render(resultInJson))
   }
+
+  private def setJobGroup(statementId: Int): String = {
```

Comment:
That will change all the related interpreter codes including `Interpreter` interface, because we need to pass in `statementId`, I would suggest to change it later on, I really don't want to change so many codes.